src/dom-builder.ts

```diff
@@ -345,8 +345,12 @@ class DomComponent<O, P, A> extends Component<O & P, A & P> {
     }
   }
   run(parent: Node, destroyed: Future<boolean>): Out<O & P, A & P> {
-    const elm = document.createElement(this.tagName);
-
+    const ns: string | undefined = (this.props as any).namespace
+    const elm: HTMLElement = ns 
+      ? (document.createElementNS(ns, this.tagName) as HTMLElement)
+      : document.createElement(this.tagName);
+    delete (this.props as any).namespace
```

Comment:
I'm beginning to think this isn't the way we want to do this... Maybe `svg` should be a proper option on `InitialProperties` and it'll set the namespace to what it's supposed to be

Reply:
It looks good to me. One advantage of having the `namespace` property is that one could use for other name spaces as well (I don't know what they would be though).